rules-reviewed/eap6/weblogic/weblogic-ejb.windup.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="weblogic-ejb" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+
+    <metadata>
+        <description>
+            This ruleset provides analysis of WebLogic proprietary classes and constructs for Enterprise Java Beans that may require
+            individual attention when migrating to JBoss EAP 6+.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,2.3.0.Final" />
+        </dependencies>
+        <sourceTechnology id="weblogic" />
+        <targetTechnology id="eap" versionRange="[6,)" />
+    </metadata>
+    <rules>
+        <rule id="weblogic-ejb-01000">
+            <when>
+                <javaclass references="weblogic.{type}" />
+            </when>
+            <perform>
+                <hint title="WebLogic proprietary EJB" effort="1" severity="mandatory">
+                    <message>
+                        This application contains WebLogic EJBs probably generated by the WebLogic Server EJB tools.
+                    </message>
+                    <link href="https://docs.oracle.com/cd/E13222_01/wls/docs81b/ejb/EJB_tools.html" title="WebLogic Server EJB Tools" />
+                    <tag>ejb</tag>
+                    <tag>weblogic</tag>
+                </hint>
+            </perform>
+            <where param="type">
+                <matches pattern="(ejb|ejb20|ejbgen)?" />
```

Comment:
Does this work properly ? I think there needs to be wildcard pattern after type parameter or in it.